jenkins/bootstrap.py

```diff
@@ -940,7 +943,11 @@ def bootstrap(args):
         logging.info('Upload result and artifacts...')
         logging.info('Gubernator results at %s', gubernator_uri(paths))
         try:
-            finish(gsutil, paths, success, '_artifacts', build, version, repos, call)
+            finish(
+                gsutil, paths, success,
+                '%s/_artifacts' % os.getenv(WORKSPACE_ENV, os.getcwd()),
```

Comment:
why not `os.path.join(os.getenv(WORKSPACE_ENV, os.getcwd()), '_artifacts')` ? :-)

Reply:
done